custom/icds_reports/views.py

```diff
@@ -1205,3 +1210,27 @@ def get(self, request, *args, **kwargs):
         return JsonResponse(data={
             'report_data': data,
         })
+
+
+@method_decorator(domain_admin_required, name='dispatch')
+class AggregationScriptPage(BaseDomainView):
```

Comment:
I think this should also use a feature flag (to be added https://github.com/dimagi/commcare-hq/blob/master/corehq/toggles.py) that way we can enable it for specific domains